src/serverdlg.cpp

```diff
@@ -123,29 +123,64 @@ CServerDlg::CServerDlg ( CServer*         pNServP,
+    pbtRecordingDir->setAccessibleName ( tr ( "Display dialog to select recording directory button" ) );
+    pbtRecordingDir->setWhatsThis ( "<b>" + tr ( "Main Recording Directory" ) + ":</b> "
+        +  tr ( "Click the button to open the dialog that allows the main recording directory to be selected."
+                "The chosen value must exist and be writeable (allow creation of sub-directories "
+                "by the user Jamulus is running as). " ) );
+
+    edtRecordingDir->setAccessibleName ( tr ( "Main recording directory text box (read-only)" ) );
+    edtRecordingDir->setWhatsThis ( "<b>" + tr ( "Main Recording Directory" ) + ":</b> "
+        +  tr ( "The current value of the main recording directory. "
+                "The chosen value must exist and be writeable (allow creation of sub-directories "
+                "by the user Jamulus is running as). "
+                "Click the button to open the dialog that allows the main recording directory to be selected." ) );
+
+    tbtClearRecordingDir->setAccessibleName ( tr ( "Clear the recording directory button" ) );
+    tbtClearRecordingDir->setWhatsThis ( "<b>" + tr ( "Clear Recording Directory" ) + ":</b> "
+        +  tr ( "Click the button to clear the currently selected recording directory. "
+                "This will prevent recording until a new value is selected.") );
+
     // enable recorder
     chbEnableRecorder->setAccessibleName( tr ( "Checkbox to turn on or off server recording" ) );
-    chbEnableRecorder->setWhatsThis( "<b>" + tr ( "Enable Recorder" ) + ":</b>"
+    chbEnableRecorder->setWhatsThis( "<b>" + tr ( "Enable Recorder" ) + ":</b> "
         + tr ( "Checked when the recorder is enabled, otherwise unchecked. "
                "The recorder will run when a session is in progress, if (set up correctly and) enabled." ) );
 
     // current session directory
     edtCurrentSessionDir->setAccessibleName( tr ( "Current session directory text box (read-only)" ) );
-    edtCurrentSessionDir->setWhatsThis( "<b>" + tr ( "Current Session Directory" ) + ":</b>"
-        +  tr ( "Enabled during recording and holds the current recording session directory. "
-                "Disabled after recording or when the recorder is not enabled." ) );
+    edtCurrentSessionDir->setWhatsThis( "<b>" + tr ( "Current Session Directory" ) + ":</b> "
+        + tr ( "Enabled during recording and holds the current recording session directory. "
+               "Disabled after recording or when the recorder is not enabled." ) );
 
     // recorder status
     lblRecorderStatus->setAccessibleName ( tr ( "Recorder status label" ) );
-    lblRecorderStatus->setWhatsThis ( "<b>" + tr ( "Recorder Status" ) + ":</b>"
-        +  tr ( "Displays the current status of the recorder." ) );
+    lblRecorderStatus->setWhatsThis ( "<b>" + tr ( "Recorder Status" ) + ":</b> "
+        + tr ( "Displays the current status of the recorder.  The following values are possible:"
```

Comment:
Sorry for the delay...
You should avoid any HTML code to be in the tr() function. That simplifies the translation and makes things more consistent. Also, in the past I have used more HTML code but after doing the German translation I found out that the more HTML code you have the more difficult is the translation. So I recommend to use as little HTML as possible. E.g. I removed the <b> and other not absolutely required HTML from the translation files.

Reply:
Argh!  Sorry, yes.  The `<dl>` was a late addition...

Reply:
See update.